src/index.html

```diff
@@ -4,9 +4,430 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header-style">
+        <a href="#">
+          <img
+            src="./images/logo.png"
+            alt="Moyo Logo"
+            class="logo"
+          >
+        </a>
+
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#apple" class="nav__link is-active">
+                Apple
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#samsung" class="nav__link">
+                Samsung
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#smartphones" class="nav__link">
+                Smartphones
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="#laptops-and-computers"
+                class="nav__link"
+                data-qa="nav-hover"
+              >
+                Laptops & computers
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#gadgets" class="nav__link">
+                Gadgets
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#tablets" class="nav__link">
+                Tablets
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#photo" class="nav__link">
+                Photo
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#video" class="nav__link">
+                Video
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main class="main">
+      <div class="main__wrapper wrapper">
```

Comment:
```suggestion
      <div class="main__container">
```

PS: `container` and `wrapper` are the same, just to stick to one naming style